src/selectors/__tests__/exportContext.ts

```diff
@@ -0,0 +1,78 @@
+import { initialState, reducerFlow } from '../../util'
+import { newThought, setCursor, importText } from '../../reducers'
+import { HOME_PATH } from '../../constants'
+import exportContext from '../exportContext'
+
+describe('exportContext', () => {
+  it('test when meta and archived thoughts are included', () => {
+    const steps = [
+      newThought({ value: 'a' }),
+      newThought({ value: 'b', insertNewSubthought: true }),
+      newThought({ value: 'c' }),
+      setCursor({ path: [{ value: 'a', rank: 0 }] })
+    ]
+
+    // run steps through reducer flow and export as plaintext for readable test
+    const stateNew = reducerFlow(steps)(initialState())
+
+    const exported = exportContext(stateNew, ['a'], 'text/plain', {
+      excludeMeta: false, excludeArchived: false
+    })
+
+    expect(exported).toBe(`- a
+  - b
+  - c`)
+  })
+
+  it('test when meta is included but archived thoughts are excluded', () => {
+    const text = `- a
+  - =pin
+    - true
+  - =archive
+    - c
+  - b`
+
+    const steps = [
+      importText({ path: HOME_PATH, text }),
+      setCursor({ path: [{ value: 'a', rank: 0 }] })
+    ]
+
+    // run steps through reducer flow and export as plaintext for readable test
+    const stateNew = reducerFlow(steps)(initialState())
+
+    const exported = exportContext(stateNew, ['a'], 'text/plain', {
+      excludeMeta: false, excludeArchived: true
+    })
+
+    expect(exported).toBe(`- a
+  - =pin
+    - true
+  - b`)
+  })
+
+  it('test when meta is excluded', () => {
+    const text = `- a
+  - b
+    - =pin
+      - true
+    - c`
+
+    const steps = [
+      importText({ path: HOME_PATH, text }),
+      setCursor({ path: [{ value: 'a', rank: 0 }] })
+    ]
+
+    // run steps through reducer flow and export as plaintext for readable test
+    const stateNew = reducerFlow(steps)(initialState())
+
+    const exported = exportContext(stateNew, ['a'], 'text/plain', {
+      excludeMeta: false, excludeArchived: true
+    })
+
+    expect(exported).toBe(`- a
+  - b
+    - =pin
+      - true
```

Comment:
If `=pin` is in the output, then it is not excluding meta thoughts as described.

Reply:
Unit test was right. The test title was wrong. :(